jetstream/experimenter.py

```diff
@@ -18,8 +18,70 @@ def _coerce_none_to_zero(x: Optional[int]) -> int:
     return 0 if x is None else x
 
 
+@attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
+class FeatureScalarTelemetry:
+    kind: str = "scalar"
+    name: str
+
+
+@attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
+class FeatureEventTelemetry:
+    kind: str = "event"
+    event_category: str
+    event_method: Optional[str]
+    event_object: Optional[str]
+    event_value: Optional[str]
+
+
+@attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
+class Feature:
+    slug: str
+    telemetry: Union[FeatureEventTelemetry, FeatureScalarTelemetry]
+
+
+@attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
+class Branch:
+    slug: str
+    ratio: int
+
+
 @attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
 class Experiment:
+    """
+    Common Experimenter experiment representation.
+    Attributes:
+        experimenter_slug: Slug generated by Experimenter for V1 experiments;
+            None for V4 experiments
+        normandy_slug: V1 experiment normandy_slug; V4 experiment slug
+        type: V1 experiment type; always "v4" for V4 experiments
+        status: V1 experiment status; "Live" for active V4 experiments,
+            "Complete" for inactive V4 experiments
+        features: empty list for V1 experiments; V4 experiment features
+        branches: V1 experiment variants converted to branches; V4 experiment branches
+        start_date: experiment start_date
+        end_date: experiment end_date
+        proposed_enrollment: experiment proposed_enrollment
+        reference_branch: V1 experiment branch slug where is_control is True;
+            V4 experiment reference_branch
```

Comment:
I added some documentation here about how the different Experiment v1 and v4 objects get converted to a general `Experiment` object